hopsandcp/CMakeLists.txt

```diff
@@ -1,19 +1,19 @@
-cmake_minimum_required(VERSION 3.1)
 project(hopsandcp)
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_DEBUG_POSTFIX _d)
 if(WIN32)
   set(CMAKE_SHARED_LIBRARY_PREFIX "")
 endif()
 
-set(CMAKE_AUTOMOC ON)
+include(${CMAKE_CURRENT_LIST_DIR}/../dependencies/dcplib.cmake)
```

Comment:
We should not include this, since DCPLib is header-only and does no longer require linking.